ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -28,6 +28,12 @@ VAV:
   - EQUIPMENT
   opt_uses:
   - zone_use_label # Needed until the zones can are joined to assets with connections.
+  connections:
+    ROOM: CONTAINS
+    AHU: FEEDS
```

Comment:
can you please update with the cardinality and this new form


```
 connections:
 AHU:
   type: FEEDS
   min: 
   max:
```

refer to the values from the sheets doc

Reply:
update to this: 
 AHU:
   type: FEEDS
   min: <number>
   max: <number>
   exact_type: <true/false>

The simple form (current) is also valid.  It is equivalent to:

AHU:
  type: FEEDS
  min: 1
  max: 1
  exact_type: false

Reply:
Done